src/logic/safe/hooks/useLoadSafe.tsx

```diff
@@ -15,7 +15,6 @@ export const useLoadSafe = (safeAddress?: string): boolean => {
 
   useEffect(() => {
     const fetchData = async () => {
-      setIsSafeLoaded(false)
```

Comment:
Wasn't this [a fix for the initial safe load](https://github.com/gnosis/safe-react/commit/baf346093e1b232a66115230fe344b13d48663ef#diff-c7124f4a6d36f8d5ad9e166103fce3bc3c5eb3d40bd5b4ede8a443ceb58733c1R18)?

Reply:
After looking into it, this line isn't necessary and is what caused the effect loop. [Sending it as a flag](https://github.com/gnosis/safe-react/commit/baf346093e1b232a66115230fe344b13d48663ef#diff-c7124f4a6d36f8d5ad9e166103fce3bc3c5eb3d40bd5b4ede8a443ceb58733c1R22) is what fixed it.

Reply:
I’m not sure about the lifecycle of this hook. Need to test loading different safes. It seems that this hook will keep the `isSafeLoaded` as true from a previous safe address, so it won't make the initial request if you have the old tags stored in localStorage.

Reply:
FYI, I've tested and it seems to still work fine. ✅